foris/locale/de/LC_MESSAGES/foris.po

```diff
@@ -555,14 +557,14 @@ msgid ""
 "password. You can then set up your mobile devices, using the QR code "
 "available next to the form."
 msgstr ""
-"Wenn Sie den Router als Wi-Fi-Zugangspunkt verwenden wollen, dann aktivieren "
-"Sie die Wi-Fi-Option. Geben Sie dann die SSID (Netzwerkname) und ein "
+"Wenn Sie den Router als WLAN-Zugangspunkt verwenden wollen, dann aktivieren "
+"Sie die WLAN-Option. Geben Sie dann die SSID (Netzwerkname) und ein "
 "dazugehöriges Kennwort ein.<br><br>Danach können Sie Ihre Mobilgeräte "
 "mittels des angezeigten QR-Codes einfach konfigurieren."
 
 #, python-format
 msgid "Enable Wi-Fi %s"
-msgstr "Aktiviere Wi-Fi %s"
+msgstr "Aktiviere WLAN %s"
```

Comment:
Just to give a popular known example, here is the German and the US help page for setting up Wi-Fi on an iPhone: DE [https://support.apple.com/de-de/HT202639](https://support.apple.com/de-de/HT202639) vs US [https://support.apple.com/en-us/HT202639](https://support.apple.com/en-us/HT202639)